logos/src/logos/main.py

```
@@ -2820,6 +2823,73 @@ async def get_process_id(data: GetProcessIdRequest):
+    """Change a user's role (for Logos Admin only)"""
+    logos_key = authenticate_logos_key(dict(request.headers))[0]
+    with DBManager() as db:
+        require_logos_admin_key(logos_key, db)
+        result, status = db.set_user_role(user_id, body.role)
+    if status != 200:
+        raise HTTPException(status_code=status, detail=result.get("error"))
+    return result
+
+@app.get("/users", tags=["users"])
+async def list_users(request: Request):
+    logos_key = require_app_admin_or_above(request)
```

Flake8 (7.3.0)

[error] local variable 'logos_key' is assigned to but never used

(F841)